.vscode/cspell.json

```diff
@@ -88,6 +88,17 @@
         "Skus",
         "Undelete"
       ]
+    },
+    {
+      "filename": "**/sdk/textanalytics/**/*.cs",
+      "words": [
+        "Brcpf",
+        "Idrg",
+        "Esdni",
+        "Eugps",
+        "Plregon",
+        "Usuk"
```

Comment:
All these are part of document names in [PiiEntityCategory](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/Models/PiiEntityCategory.cs).

Reply:
oh wow. no idea what they are :|